### PR TITLE
[#2814][#2688] improve(web): add configuration file to manage common user interface variables

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

Add configuration file to manage common user interface variables.
`web/src/configs/themeConfig.js`

1. `toastDuration`: Modified toast message box duration to `4000` milliseconds.

Because the previous error message box wouldn't disappear on its own, in order to avoid confusing the users, we made the error message box disappear automatically after 4 seconds.

2. `toastPosition`: Modified the message box position, the default is `top-right`.

### Why are the changes needed?

Fix: #2814
Fix: #2688

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

File: web/src/components/StyledToast.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function StyledToast() {
         reverseOrder={false}
         position='top-right'
         toastOptions={{
-          duration: Infinity,
+          duration: 4000,
```
Comment (zhaoyongjie) [inline web/src/components/StyledToast.js:17]: Hey @ch3yne , we should define a constant value for the debounce value for the persistent UX. Thanks!

Comment (ch3yne) [reply web/src/components/StyledToast.js:17]: Users usually don't modify the frontend code, do you means that we need to add this in the Gravitino configuration file?

Comment (zhaoyongjie) [reply web/src/components/StyledToast.js:17]: Nope, it should be defined in a constant file and referred to. IMO, debounce is common and frequent in UI.

Comment (ch3yne) [reply web/src/components/StyledToast.js:17]: Okay, I will add a configuration file to manage common user interface variables.

Comment (zhaoyongjie) [reply web/src/components/StyledToast.js:17]: @ch3yne Great! Thanks.

**Review by zhaoyongjie — APPROVED** [review]
Thanks for the improvement. LGTM.

**Review by LauraXia123 — APPROVED** [review]

**Pull request merged**